api/rpc/account/account.proto

```diff
@@ -146,7 +146,8 @@ message SignUpRequest {
 
 message VerificationRequest {
   string name = 1;
-  string code = 2;
+  string password = 2;
```

Comment:
Ah, good point. We can't specify the length of either bytes or strings in the protobuf definion, only validate it at the receiver.